Quick Pad/MultilingualResources/Quick Pad.fr-FR.xlf

```diff
@@ -0,0 +1,700 @@
+          <source>Classic Mode</source>
+          <target state="translated">Mode classique</target>
+        </trans-unit>
+        <trans-unit id="CmdExit.Label" translate="yes" xml:space="preserve">
+          <source>Exit</source>
+          <target state="translated" state-qualifier="tm-suggestion">Quitter</target>
+        </trans-unit>
+        <trans-unit id="CmdExitTooltip.Text" translate="yes" xml:space="preserve">
+          <source>Exit the app</source>
+          <target state="translated">Quitter l'application</target>
+        </trans-unit>
+        <trans-unit id="ClassicFormatFonts.Text" translate="yes" xml:space="preserve">
+          <source>Fonts...</source>
+          <target state="translated">Polices...</target>
+        </trans-unit>
+        <trans-unit id="ClassicViewStatusBar.Text" translate="yes" xml:space="preserve">
+          <source>Status Bar</source>
+          <target state="translated" state-qualifier="tm-suggestion">Barre d'état</target>
+        </trans-unit>
+        <trans-unit id="FAF_Color.Text" translate="yes" xml:space="preserve">
+          <source>Color</source>
+          <target state="translated" state-qualifier="tm-suggestion">Couleur</target>
+        </trans-unit>
+        <trans-unit id="FAF_Fonts.Text" translate="yes" xml:space="preserve">
+          <source>Fonts</source>
+          <target state="translated" state-qualifier="tm-suggestion">Polices</target>
+        </trans-unit>
+        <trans-unit id="FAF_Preview_Initial" translate="yes" xml:space="preserve">
+          <source>Preview</source>
+          <target state="translated" state-qualifier="tm-suggestion">Aperçu</target>
+        </trans-unit>
+        <trans-unit id="FAF_Size.Text" translate="yes" xml:space="preserve">
+          <source>Size</source>
+          <target state="translated" state-qualifier="tm-suggestion">Taille</target>
+        </trans-unit>
+        <trans-unit id="StatusColCount.Text" translate="yes" xml:space="preserve">
+          <source>Column: </source>
+          <target state="translated">Colonne:</target>
+        </trans-unit>
+        <trans-unit id="StatusLnCount.Text" translate="yes" xml:space="preserve">
+          <source>Line: </source>
+          <target state="translated">Ligne:</target>
+        </trans-unit>
+        <trans-unit id="StatusSelCount.Text" translate="yes" xml:space="preserve">
+          <source>Selection: </source>
+          <target state="translated">Sélection:</target>
+        </trans-unit>
+        <trans-unit id="GeneralShowStatusOnClassic.Text" translate="yes" xml:space="preserve">
+          <source>Show Status Bar on Classic Mode</source>
+          <target state="translated">Montrer la barre d'état en mode classique</target>
+        </trans-unit>
+        <trans-unit id="AppLanguage.Text" translate="yes" xml:space="preserve">
+          <source>App Language</source>
+          <target state="translated">Langue de l'application</target>
+        </trans-unit>
```

Comment:
Not sure where you've got a translation from, but this one is older than the current version.
This text resource is still on App Language, but we currently have about 10-20 more text after this